packages/cloudflare/src/cli/templates/images.ts

```diff
@@ -0,0 +1,90 @@
+export type RemotePattern = {
+  protocol?: "http" | "https";
+  hostname: string;
+  port?: string;
+  pathname: string;
```

Comment:
I believe pathname can be optional here now that we're no longer setting it to a fallback, or does this get defaulted in the manifest?

Reply:
Good question!
It's actually an always set regexp in the manifest - I'll add a comment